nvme-print-stdout.c

```diff
@@ -1410,7 +1426,7 @@ static void stdout_registers_pmrebs(__u32 pmrebs)
 static void stdout_registers_pmrswtp(__u32 pmrswtp)
 {
 	printf("\tPMR Sustained Write Throughput       (PMRSWTV): %x\n",
-		(pmrswtp & 0xffffff00) >> 8);
+	       (pmrswtp & 0xffffff00) >> 8);
```

Comment:
Again, don't mix random whitespace changes with new or refactored code. It's noise. Also I think the code base looks in reasonable shape thus we should really stop with this type of code churn anyway.